charts/nginx-ingress-controller/templates/service.yaml

```diff
@@ -26,6 +26,9 @@ metadata:
     {{- toYaml .Values.nginx.service.annotations | nindent 4 }}
 spec:
   type: LoadBalancer
+{{- if .Values.nginx.service.loadBalancerIP }}
+  loadBalancerIP: {{ .Values.nginx.service.loadBalancerIP }}
```

Comment:
IMHO put the IP in quotes, just in case :)

Reply:
And you can avoid repeating the key by doing

```
{{- with .Values.nginx.service.loadBalancerIP }}
  loadBalancerIP: {{ . | quote }}
{{- end }}
```

Reply:
I just copied [same code from upstream](https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/templates/controller-service.yaml#L25-L27) so that it's easier to maintain, if the need arises later to match upstream chart. 

If you still insist, I can make that change.. but it seems nitpicking to me ATM.

Reply:
Ok, the IP was quoted in older version of the charts, now it isn't so perhaps it is safe ;-) I'll resolve and lgtm